README.md

```diff
@@ -1 +1,27 @@
-# tool-template
\ No newline at end of file
+#  BIOPET tool suite
+This tool is part of BIOPET tool suite that is developed at LUMC by [the SASC team](http://sasc.lumc.nl/). 
+Each tool in the BIOPET tool suite is meant to offer a standalone function that can be used to perform a
+dedicate data analysis task or added as part of [BIOPET pipelines](http://biopet-docs.readthedocs.io/en/latest/).
+
+#  About this tool
+<Description of a tool>
+
+#  Installation
+This tool requires Java 8 to be installed on your device. Download Java 8 
+[here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html) 
+or install via your distribution's package manager.
+
+<How to download and run a tool>
+
+#  Manual
+<Help doc of a tool>
+
+#  Contact
+
+
+<p>
+  <!-- Obscure e-mail address for spammers -->
+For any question related to this tool, please use the github issue tracker or contact 
+  <a href='http://sasc.lumc.nl/'>the SASC team</a> directly at: <br>
+  &#115;&#97;&#115;&#99;&#32;&#91;&#97;&#116;&#93;&#32;&#108;&#117;&#109;&#99;&#32;&#91;&#100;&#111;&#116;&#93;&#32;&#110;&#108;
```

Comment:
@rhpvorderman: This show a bit weird not, not possible to just to sasc@lumc.nl with a mailto link?